demo/src/index.html

```diff
@@ -62,21 +62,11 @@
             <section><div id="app-root"></div></section>
             <footer>
                 <a href="https://www.springload.co.nz/" style="display: block;">
-                    <svg
-                        xmlns="http://www.w3.org/2000/svg"
-                        viewBox="0 0 160 28"
-                        style="width:144px; height:24px;"
-                    >
-                        <title>Springload</title>
-                        <path
-                            fill="#4A4A4A"
-                            d="M8.9 17.7c0-1-.9-1.1-2.9-1.3-2.6-.2-5.4-.7-5.4-4.1C.6 9.1 3.7 8 6.4 8c4 0 5.5 1.3 5.9 4.2l-3.6.4c-.2-1.2-.8-1.8-2.5-1.8-1.1 0-2.1.3-2.1 1.2s.8 1 2.7 1.2c2.7.2 5.5.6 5.5 4.1 0 3.3-2.9 4.4-6.2 4.4-4 0-5.9-1.4-6.1-4.3l3.6-.4c.1 1.3.9 1.9 2.6 1.9 1.7.1 2.7-.1 2.7-1.2m8.6 8.8h-3.4V11.8l-.2-3.6H17l.2 1.7c.7-1.4 2.4-2 4-2 4.1 0 6.1 2.4 6.1 7s-2.1 7-6.1 7c-2 0-3.2-.7-3.8-1.7v6.3h.1zm3.1-15.4c-1.8 0-2.9 1.1-3.2 2.2v3.1c.2 1.1 1.4 2.2 3.2 2.2 1.8 0 3.1-1.3 3.1-3.9 0-2.4-1.3-3.6-3.1-3.6m16.6.2c-2.6 0-4.1 1.6-4.5 2.8v7.4h-3.3v-9.7l-.2-3.6h3.1l.2 2.4c.7-1.6 2-2.9 4.8-2.9v3.6h-.1zm3.4-10.2c1.2 0 2.3 1 2.3 2.3 0 1.2-1 2.2-2.3 2.2-1.2 0-2.2-1-2.2-2.2-.1-1.2 1-2.3 2.2-2.3m-1.8 7.1h3.4v13.3h-3.4V8.2zm9 1.7c.8-1.2 2.1-2.1 4.1-2.1 3.1 0 4.9 1.6 4.9 4.7v9h-3.4V13c0-1.3-.7-2-2.1-2-1.8 0-3 1.2-3.3 2.2v8.2h-3.3v-9.6l-.1-3.6h3.1l.1 1.7zm20.7 9.5c-.6 1.2-2 1.7-3.8 1.7-3.9 0-6.1-2.1-6.1-6.7s2.3-6.7 6.2-6.7c1.9 0 3.2.7 3.9 2l.2-1.6H72l-.1 3.6v9.8c0 3.7-2.8 5.3-6.5 5.3-4.4 0-6.1-1.6-6.3-4.2l3.7-.6c.1 1 .6 1.9 2.6 1.9 1.9 0 3.2-.8 3.2-2.7l-.1-1.8zM65.2 18c1.8 0 3-1.1 3.2-2.3v-2.3c-.2-1.2-1.4-2.4-3.2-2.4-1.8 0-3.1.9-3.1 3.6 0 2.5 1.4 3.4 3.1 3.4m9.1-16.5h3.4v20.1h-3.4V1.5zm5.3 13.4c0-4.7 2.6-7 6.5-7 4 0 6.5 2.3 6.5 7s-2.6 7-6.5 7c-4 0-6.5-2.5-6.5-7m3.5 0c0 2.8 1.1 3.8 3 3.8s3-1 3-3.8-1.1-3.8-3-3.8-3 1-3 3.8m24.2-3.1v6.1l.2 3.6h-3.1l-.2-1.6c-.6 1.3-1.9 1.9-3.8 1.9-4 0-6.1-2.4-6.1-7s2.2-7 6.1-7c2 0 3.1.8 3.8 1.9l.2-1.4h3.1l-.2 3.5zm-3.4 1.5c-.2-1.1-1.4-2.2-3.2-2.2-1.8 0-3 1.2-3 3.8s1.2 3.8 3 3.8 3-1.1 3.2-2.2v-3.2zm18.6-11.8v16.4l.2 3.6h-3.1l-.2-1.6c-.6 1.2-1.9 1.9-3.9 1.9-4 0-6.1-2.4-6.1-7s2.2-7 6.1-7c1.8 0 3.1.6 3.7 1.7v-8h3.3zm-3.3 11.8c-.2-1.1-1.4-2.2-3.2-2.2-1.8 0-3 1.2-3 3.8s1.2 3.8 3 3.8 3-1.1 3.2-2.2v-3.2z"
-                        ></path>
-                        <path
-                            fill="#7DC265"
-                            d="M154.7 8.8c-3.3 0-4.4-1-7.9-1-3 0-5.9 1.1-6.4 6.1-.4 3.7-2.2 5-4.2 5-1.6 0-2.7-.6-3.1-1.8 2-.7 3.1-2.3 3.1-4.5 0-2.8-1.8-4.8-4.9-4.8-3.1 0-4.9 2-4.9 4.8 0 2.3 1.2 4 3.3 4.7.8 3.3 3.2 4.5 6.4 4.5 3.8 0 6.7-2.3 7.3-6.8 1.6 1.7 3.6 2.1 5.5 2.1 5.7 0 9.8-4.3 11-9.4-1.8.8-3.3 1.1-5.2 1.1zm-23.5 5.9c-1.1 0-1.8-.8-1.8-1.9s.7-1.9 1.8-1.9c1.2 0 1.8.8 1.8 1.9-.1 1-.6 1.9-1.8 1.9z"
-                        ></path>
-                    </svg>
+                    <img
```

Comment:
Using an `<img>` instead of SVG because the svg file has a lot of details and hard to look at.